contracts/Auction.sol

```diff
@@ -143,28 +127,63 @@ contract Auction is IAuction {
         return self.amountOutstanding > 0;
     }
 
+    /// @notice Takes an offer from an auction buyer with a minimum required tokens
+    ///         to seize from the coverage pool.
+    /// @dev 'minAmount' sets a minimum amount of tokens to seize in this transaction.
+    ///      If a minimum amount is not satisfied, then transaction will revert.
+    /// @param amount the amount the taker is paying, denominated in tokenAccepted
+    /// @param minAmountToSeize minimum amount of tokens to seize from the coverage pool
+    function takeOfferWithMin(uint256 amount, uint256 minAmountToSeize)
```

Comment:
Wouldn't it be better/cleaner to rip it out of the `Auction` contract entirely? This feels like a utility function so we can keep it in `AuctionBidder(Utils)` contract or something similar where we'd inspect the coverage pool balance and compute if it is enough? We would not have to go through the entire hierarchy of components.

Reply:
Okay, I was on the fence with having two places where an offer can be taken, but added `AuctionBidder` contract and moved `takeOfferWithMin` there with new logic 11cc048